http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -539,61 +539,42 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
           config.deleteVmPollConfig.interval
         ).compile.lastOrError
 
-        continue = for {
-          diskResourceOpt <- controlledResourceQuery
-            .getWsmRecordForRuntime(runtime.id, WsmResourceType.AzureDisk)
-            .transaction
-          _ <- logger
-            .info(ctx.loggingCtx)(
-              s"No disk resource found for delete azure runtime msg $msg. No-op for wsmDao.deleteDisk."
-            )
-            .whenA(diskResourceOpt.isEmpty)
-          deleteDisk = diskResourceOpt.traverse { disk =>
-            wsmDao.deleteDisk(
-              DeleteWsmResourceRequest(
-                msg.workspaceId,
-                disk.resourceId,
-                DeleteControlledAzureResourceRequest(
-                  WsmJobControl(WsmJobId(s"delete-disk-${ctx.traceId.asString.take(10)}"))
-                )
-              ),
-              auth
-            )
-          }.void
-
-          networkResourceOpt <- controlledResourceQuery
```

Comment:
Why was this deleted? Should we keep handling network deletion?

Reply:
this was in response to Qi's comment here: https://github.com/DataBiosphere/leonardo/pull/3095#discussion_r1067357873

Reply:
Gotcha, sorry for missing it 😵‍💫